### ISSUE TITLE
what's your original learning rate? on my training，the loss is in plateau.

### ISSUE THREAD

**Issue by Sierkinhane** [last-one/pytorch_realtime_multi-person_pose_estimation#35]

**Comment by Sierkinhane** [comment]
Can you show me your train loss and val loss ？plz
